jvmhs/src/Jvmhs/Data/Code.hs

```diff
@@ -158,9 +165,423 @@ verificationTypeInfo g (B.StackMapTable s) =
 $(deriveToJSON defaultOptions{fieldLabelModifier = camelTo2 '_' . drop 5} ''Code)
 $(deriveToJSON defaultOptions{fieldLabelModifier = camelTo2 '_' . drop 3} ''ExceptionHandler)
 
-instance ToJSON (B.ByteCodeOpr B.High) where
-  toJSON _ = String "<code>"
+instance ToJSON ByteCodeOpr where
+  toJSON = \case
+    B.ArrayLoad arrType -> object
+      [ "opc" .= String "array_load"
+      , "type" .= toJSON arrType
```

Comment:
You do not need toJSON infront of arrType. 

If you look at the definition of [(.=)](http://hackage.haskell.org/package/aeson-1.4.0.0/docs/Data-Aeson.html#g:18)

```
(.=) :: ToJSON v => Text -> v -> kv 
```

This means that if v is a ToJSON you can hand it ind directly.

```suggestion
      , "type" .= arrType
```